app/models/school_group.rb

```diff
@@ -1,4 +1,6 @@
 class SchoolGroup < Organisation
   has_many :school_group_memberships
   has_many :schools, through: :school_group_memberships
+
+  scope :trusts, -> { where.not(uid: nil) }
```

Comment:
this scope is available from the super class

Reply:
🤦